tradingstrategy/utils/groupeduniverse.py

```diff
@@ -59,6 +59,7 @@ def __init__(self,
         self.pairs: pd.GroupBy = self.df.groupby(["pair_id"])
         self.timestamp_column = timestamp_column
         self.time_bucket = time_bucket
+        self.tolerance_time_bucket = TimeBucket
```

Comment:
We probably do not need it here, because we can have it as the parameter for `BacktestPriceModel` class in `trade-executor`